tests/scripts/all.sh

```diff
@@ -3594,6 +3594,88 @@ component_test_psa_crypto_config_accel_aead () {
     make test
 }
 
+# This is a common configuration function used in:
+# - component_test_psa_crypto_config_accel_cipher_aead
+# - component_test_psa_crypto_config_reference_cipher_aead
+common_psa_crypto_config_accel_cipher_aead() {
+    scripts/config.py -f "$CRYPTO_CONFIG_H" unset PSA_WANT_ALG_CMAC
```

Comment:
Actually, after #8314 I think it would be better to accelerate CMAC than to disable it. (In the general spirit of disabling on the minimum.) So:
- don't unset `ALG_CMAC` or PBKDF2-CMAC  here but add `ALG_CMAC` to `loc_accel_list` in the accel component;
- don't unset `CMAC_C` here but only in the accel component;
- ignore `test_suite_cmac` in `analyze_outcomes.py`.

(This will need rebasing on recent development.)